packages/ibm-products/src/components/Cascade/Cascade.docs-page.js

```diff
@@ -8,19 +8,18 @@
 import React from 'react';
 import { StoryDocsPage } from '../../global/js/utils/StoryDocsPage';
 import * as stories from './Cascade.stories';
-import { useOf } from '@storybook/blocks';
-import { storyDocsGuidelines } from '../../global/js/utils/story-helper';
 
 const DocsPage = () => {
-  const { csfFile } = useOf('meta', ['meta']);
-
   return (
     <StoryDocsPage
       altGuidelinesHref={[
-        storyDocsGuidelines(csfFile),
         {
-          href: 'https://www.carbondesignsystem.com/guidelines/motion/overview/',
-          label: 'Carbon motion guidelines',
+          href: 'https://pages.github.ibm.com/carbon/ibm-products/patterns/cascade/',
+          label: 'Cascade overview',
```

Comment:
Interesting that we’re calling it “overview” instead of usage guidelines. Is it because the Cascade doesn’t go to a Usage tab? But 
I would keep with the same naming for now but open this as a discussion in #6959.

```suggestion
          label: 'Cascade usage guidelines',
```

Reply:
That's the problem with specificity.

`/ibm-products/components/about-modal/usage/` → "AboutModal usage guidelines"
- Is part of the "Component" navigation.
- "/usage/" in the URL.
- "Usage" tab name.
- "guidelines" is a word used in the doc, but not specifically about AboutModal.


`ibm-products/patterns/cascade/`
- Is part of the "Patterns" navigation.
- No mention of "/usage/" in the URL.
- No "Usage" tab. (No tabs at all)
- No mention of "guidelines" anywhere.
- "Overview" is the first section. (But is one of three with the same heading level)

`ibm-products/guidelines/accessibility/overview/`
- "Guidelines" is a specific section in the navigation. (But it's not directly linkable)

Reply:
While I understand your point, for this current issue, I’d avoid introducing new naming until we have a larger discussion on the IA via #6959.

For most of our patterns, you’ll notice that the majority of our “patterns” do still have a `Usage` and `Code` tab, whether or not that’s correct. (You’ll see occasional ”Overviews” too.) While I’m not sure why Cascade and Notifications panel don’t currently have one, I would treat them the same until the larger docs issue.